src/server/dashboard/index.ts

```diff
@@ -154,7 +175,30 @@ function parseWorkspaces(bundles: NodeCG.Bundle[]): Workspace[] {
 		});
 	});
 
-	workspaces.sort((a, b) => a.label.localeCompare(b.label));
+	// Sort workspaces by order first, then alphabetically
+	workspaces.sort((a, b) => {
+		const orderA = workspaceOrders.get(a.name);
+		const orderB = workspaceOrders.get(b.name);
+
+		// Both have orders: sort by order, then alphabetically if same order
+		if (orderA !== undefined && orderB !== undefined) {
+			if (orderA !== orderB) {
+				return orderA - orderB;
+			}
+			// Same order: fall through to alphabetical sorting
+		}
+		// Only A has order: A comes first
+		else if (orderA !== undefined) {
+			return -1;
+		}
+		// Only B has order: B comes first
+		else if (orderB !== undefined) {
+			return 1;
+		}
+
+		// Both have same order or neither has order: sort alphabetically
+		return a.label.localeCompare(b.label);
+	});
```

Comment:
sometimes there are use cases where you want to put a certain workspace or fullbleed panel to the last. we can do that by naming it like `X-Misc-DoNotTouch` or something and be sorted alphabetically. I wonder if it can be more explicitly configurable with this order system. It doesn't have to be implemented in this PR but would be nice to have the design that is extendable.